frontend/src/components/VContentPage.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <div class="pt-5 md:pt-10" dir="ltr">
+  <VSkipToContentContainer as="main" class="pt-5 md:pt-10" dir="ltr">
```

Comment:
```suggestion
  <VSkipToContentContainer as="main" class="pt-5 md:pt-10 flex-grow" dir="ltr">
```

This fixes a small visual bug on http://localhost:8443/privacy, where the main element is shorter than the available space so you see the focus border in the middle of the screen:

![CleanShot 2023-02-22 at 09 20 57](https://user-images.githubusercontent.com/6351754/220651284-b136ea0a-4480-47c3-9e91-ea79b57973be.png)